examples/Basic/console.py

```diff
@@ -24,13 +24,13 @@
 from pymeasure.display.Qt import QtWidgets
 from pymeasure.display.windows import ManagedWindow
 import logging
+
 log = logging.getLogger('')
 log.addHandler(logging.NullHandler())
 
 
 class TestProcedure(Procedure):
-
-    iterations = IntegerParameter('Loop Iterations', default=100)
+    iterations = IntegerParameter('Loop Iterations', default=10)
```

Comment:
I think this change is not really needed. Right ?

Reply:
Nope, I had lowered it for faster testing. Increased to 100 in last commit.